user-guide-chapters/introduction/launching.adoc

```diff
@@ -36,18 +43,22 @@ A simple JSON configuration file is shown in the following example.
 }
 ----
 
-The first attribute of the JSON data is the distribution specification version
+The configuration file contains the distribution specification version
 (`distSpecVersion`). The structure and content of other attributes are described
 in the later sections of this guide.
 
-Using the information in this guide, compose a configuration file with the
+Using the information in this guide, you can compose a configuration file with the
 settings and features you require for your {zotLowerName} registry server.
+
+
+=== Verifying the Configuration and Launching {zotLowerName}
+
 Before deploying {zotLowerName}, verify the syntax of your configuration file by
```

Comment:
This protects against operator errors and also configuration changes that may have occurred across zot versions.

Reply:
"The verification protects against operator errors and any conflicts arising from zot version changes."